content/v2.0/security/enable-tls.md

```diff
@@ -13,44 +13,61 @@ v2.0/tags: [security, authentication, tls, https, ssl]
 Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
 When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
 
+This pages outlines how to set up TLS over HTTPS with InfluxDB using either a signed or self-signed certificate.
```

Comment:
To set up TLS over HTTPS, do the following:
- Obtain [requirements](#requirements)
- [Set up InfluxDB....](#set-up...)

Reply:
I'm thinking of going with "Configure InfluxDB to use TLS"—